File: test/fixtures/node-axios/get_with_data.js
import axios from 'axios';

const response = await axios.get('http://localhost:28139/synthetics/api/v3/monitors', {
  params: {
    'test': '2',
    'limit': '100',
    'w': '4'
  },
  headers: {
    'X-Api-Key': '123456789'
  }
});

File: test/fixtures/objectivec/get_with_data.m
#import <Foundation/Foundation.h>

NSURL *url = [NSURL URLWithString:@"http://localhost:28139/synthetics/api/v3/monitors?test=2&limit=100&w=4"];
NSDictionary *headers = @{
    @"X-Api-Key": @"123456789"
};

NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:url];
[request setAllHTTPHeaderFields:headers];

NSURLSessionConfiguration *defaultSessionConfiguration = [NSURLSessionConfiguration defaultSessionConfiguration];
NSURLSession *session = [NSURLSession sessionWithConfiguration:defaultSessionConfiguration];
NSURLSessionDataTask *dataTask = [session dataTaskWithRequest:request completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
    if (error) {
        NSLog(@"%@", error);
    } else {
        NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *) response;
        NSLog(@"%@", httpResponse);
    }
}];
[dataTask resume];
